docs/glossary.md

```diff
@@ -0,0 +1,202 @@
+---
+id: glossary
+title: Cumulus Glossary
+hide_title: true
+---
+
+# AWS Glossary
+
+For terms/items from Amazon/AWS not mentioned in this glossary, please refer to the [AWS Glossary](https://docs.aws.amazon.com/general/latest/gr/glos-chap.html).
+
+# Cumulus Glossary of Terms
+
+### API Gateway
+
+  A Cumulus component that provides an API that provides accessors to Cumulus functionality.
```

Comment:
API Gateway is really an AWS thing. Cumulus uses API gateway for its API. https://aws.amazon.com/api-gateway/

Reply:
🤔  Right, in the context of this doc I believe there was at least one reference to the Cumulus API Gateway that seemed to overload the term.   In either case we should probably re-write this a bit to highlight that 👍

Reply:
03e1647c